rackcity/views/it_model_views.py

```diff
@@ -97,15 +97,16 @@ def model_modify(request):
 def model_height_change_valid(new_model_data, existing_model):
     if 'height' not in new_model_data:
         return True
-    if new_model_data['height'] <= existing_model.height:
+    new_model_height = int(new_model_data['height'])
```

Comment:
Same issue as above. We don't use a serializer to validate the data on bulk upload responses because not all required fields are necessary. So this isn't automatically casted to an int yet

Reply:
Me